CodeReview/src/commands/CCodeReview.cpp

```diff
@@ -51,6 +51,9 @@ using namespace Visualization;
 
 namespace CodeReview {
 
+const QString CCodeReview::SAVE_COMMAND = "save";
```

Comment:
It is actually part of the review command. So to execute it you have to write `review save`.
Is that OK?